core/src/test/java/org/apache/calcite/test/RelOptRulesTest.java

```diff
@@ -2188,7 +2188,7 @@ public void testMinusMergeRule() throws Exception {
 
   /** Test case for
    * <a href="https://issues.apache.org/jira/browse/CALCITE-3198">[CALCITE-3198]
-   * Enhance RexSimplify to handle (x<>a or x<>b)</a>. */
+   * Enhance RexSimplify to handle (x!=a or x!=b)</a>. */
   @Test public void testReduceConstantsDup3Null() throws Exception {
```

Comment:
Why we use the `!=` which is not a sql operator ?

Reply:
It was just a quick fix, since the usage of "<>" broke the javadoc:
```
[ERROR]    .../apache/calcite/test/RexProgramTest.java:1532: error: bad use of '>'
[ERROR]    * Enhance RexSimplify to handle (x<>a or x<>b)</a>. */
[ERROR]                                               ^
```
It is just the javadoc of a unit test, I think using "<>" or "!=" was not very relevant.

Reply:
i think we  should use the "& lt ;" token instead.

Reply:
Ok, I can change it into: 
`Enhance RexSimplify to handle (x&lt;&gt;a or x&lt;&gt;b)`
IMHO that would impact the clarity of the javadoc comment, but I'm not against it.

Reply:
fixing it via https://github.com/apache/calcite/pull/1368